### ISSUE TITLE
Doubt regarding dataset

### ISSUE THREAD

**Issue by Shekharmeena28** [Mostafa-Nafie/Football-Object-Detection#1]
Hey there, i hope you are doing okay, i just want to ask as the soccer net dataset is jumbled up, each and every events of match like yellow card , red card, goals, free kick are in different folder with regards to match, how did u put them in single folder train and test

**Comment by Mostafa-Nafie** [comment]
Hello, the soccernet dataset I used wasn't meant for event detection or classification, it was meant for **object detection**. It contained the ground truth of the detected objects including the detections with their corresponding bounding boxes. It was divided into train/val/test folders, with each one of them containing images/labels folders, which is the expected format for yolov8 training dataset.

**Comment by Shekharmeena28** [comment]
as the dataset is tedious and it is not in yolov8 format , so i am confused how did u do it, as yolov8 requires txt format for every image which contains the bounding box and class name and this here soccernet is json format

**Comment by Mostafa-Nafie** [comment]
As I have mentioned before, the dataset that I used was already provided in the required format.
